cmd/metal-api/internal/grpc/grpc-server.go

```diff
@@ -130,16 +130,19 @@ func Run(cfg *ServerConfig) error {
 				return false
 			}
 		}
-		auditStreamInterceptor, err := auditing.StreamServerInterceptor(cfg.Auditing, log.WithGroup("auditing-grpc"), shouldAudit)
-		if err != nil {
-			return err
-		}
-		auditUnaryInterceptor, err := auditing.UnaryServerInterceptor(cfg.Auditing, log.WithGroup("auditing-grpc"), shouldAudit)
-		if err != nil {
-			return err
+
+		for _, backend := range cfg.Auditing {
```

Comment:
here we need an if statement to skip not enabled backends. See https://github.com/metal-stack/metal-roles/pull/310/files#diff-81777600d83d4794dafee8e817a4b696b9c1a05fe38798329235b2ec72537cd1R229

Reply:
The audit backends are not appended to the slice in case they are not enabled, so I think there is no need to skip here.

Reply:
It's a good idea to not append disabled backends, though I think it's confusing to have an `Enabled` field in Go but not using it. Could be either remove enabled from the struct or implement the skip here?